src/index.js

```diff
@@ -8,24 +8,32 @@ import Share from './share';
 const PRISMIC_ENDPOINT = window.prismic && window.prismic.endpoint;
 let START_EXPERIMENT = () => {};
 
-const config = (() => {
+export const config = (() => {
   const matches = PRISMIC_ENDPOINT && PRISMIC_ENDPOINT.match(new RegExp('(https?://([^/]*))'));
   if (matches && matches[1] && matches[2]) {
     const baseURL = matches[1].replace(/\.cdn\.prismic\.io/, '.prismic.io');
+    const corsLink = iFrame(`${baseURL}/previews/messenger`)
     const editorTab = matches[2].replace(/\.cdn\.prismic\.io/, '.prismic.io');
     const location = {
       origin: window.location.origin,
       hash: window.location.hash,
       pathname: window.location.pathname,
       search: window.location.search,
     };
-    return { baseURL, editorTab, location };
+    return { baseURL, editorTab, location, corsLink };
   }
   return null;
 })();
 
 const setupToolbar = Utils.debounce(() => Toolbar.setup(), 500, true);
 
+function iFrame(src) {
+  const iframe = document.createElement('iframe')
+  iframe.src = src
+  document.head.appendChild(iframe)
```

Comment:
Weird the iframe is appended to the `head` and not to the `body`.

Reply:
I'd prefer you put this code in `share.js` for now.

Reply:
Replying to your first comment: It's in the `head` because it doesn't get rendered and this JS runs before the `body` loads (it would error if I appended to `body`)